src/docs/product/performance/transaction-summary.mdx

```diff
@@ -103,7 +103,13 @@ The "Spans" tab displays a list of suspect spans that correspond to where most o
 
 Choose from one of several [metrics](#metrics) to sort spans and identify different types of problems. Filter spans to only see the span operation type you're interested in.
 
-Clicking on a span will give you more details about the specific span [group](#grouping). You can see the performance of the span over time and see a list of transaction events that contain the span.
+Clicking on a span will take you to its summary page that should give you more details about this specific span [group](#grouping). On the page you can see the performance of the span over time and see a list of transaction events that contain the span.
+
+<Note>
+
+If you're in the Early Adopter program, you can also see the distribution of the span self time. The self time histogram can help you identify systemic performance issues accross the entire dataset by illustrating how self time for every span instance is distributed. You can also select a subset of the dataset by zooming into the distribution histogram which will effectively set min and max self time bounds. The stats and transaction events table will reflect that change and will only include spans with self time that are within the specified bounds. Adding self time filters allows you to narrow down your focus on the best/worse performing spans so you can troubleshoot performance issues easier.
```

Comment:
```suggestion
If you're in the Early Adopter program, you can also see the distribution of the span self time. The histogram can help you identify systemic performance issues across the entire dataset by illustrating how self time for every span instance is distributed. You can also select a subset of the dataset by zooming into the distribution histogram which will effectively set min and max bounds. The stats and transaction events table will reflect that change and will only include spans with self time that are within the specified bounds. Adding filters allows you to narrow down your focus on the best/worse performing spans so you can troubleshoot performance issues easier.
```

I feel like we've said self time (self-time?) a lot here. I removed a couple of instances but may need to play around with where we include it.

Reply:
This reads better for sure. Yeah I've never written self time so much in a single paragraph 😅